src/app/pages/component-category-list/component-category-list.ts

```diff
@@ -26,6 +28,16 @@ export class ComponentCategoryList implements OnInit {
     this.params = Observable.combineLatest(
       this._route.pathFromRoot.map(route => route.params),
       Object.assign);
+
+    // title on topbar navigation
+    this.routeParamSubscription = this._route.parent.parent.params.subscribe( params => {
+      let sectionName = params['section'];
+      this._componentPageTitle.title = SECTIONS[sectionName];
+    });
```

Comment:
Try this, it's a little simpler

```ts
this.routeParamSubscription = this.params.subscribe(params => {
  const sectionName = params['section'];
  this._componentPageTitle.title = SECTIONS[sectionName];;
});
```

Reply:
Thanks @willshowell, implemented the changes. Please review.